persistence/ravendb/license.md

```diff
@@ -0,0 +1,23 @@
+---
+title: RavenDb License
+component: raven
+tags:
+ - Persistence
+reviewed: 2020-04-06
+---
+[RavenDb](https://ravendb.net/) is a choice as one of several persisters to store NServiceBus specific information.
+
+[Hibernating Rhinos Ltd](https://hibernatingrhinos.com/) and [Particular Software Ltd](https://particular.net/) have reached an agreement whereby all NServiceBus users are granted a license to use RavenDB for NServiceBus' purposes and such use of RavenDB will be governed by the NServiceBus license agreement.
+
+You may
+
+- Install RavenDB for the purpose of storing NServiceBus on the same number of cores/machines as your NServiceBus license.
```

Comment:
Obviously this and the next item in this list will soon be an issue.

Reply:
Just discussed this with @WilliamBZA. We'll do this PR as is and William will take care of updating this section soonish.

Reply:
I've added it to our launch issue. We'll make sure it gets handled.